### PR TITLE
Fix some excessive looking resource asks and one erroneous entry

### PR DESCRIPTION
**Pull request by pidster**
Fixes https://github.com/microservices-demo/microservices-demo/issues/753

### CODE REVIEW

**Comment by pidster** [comment]
@idcrosby please take a look.

File: deploy/kubernetes/manifests/payment-dep.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ spec:
             cpu: 100m
             memory: 100Mi
           requests:
-            cpu: 101m
+            cpu: 99m
```
Comment (idcrosby) [inline deploy/kubernetes/manifests/payment-dep.yaml:24]: The 101 was just to test the CI around config. I had meant to revert it.  But why 99?

**Review by idcrosby — APPROVED** [review]
:shipit:

**Comment by errordeveloper** [comment]
This is good, but please note that setting `JAVA_OPTS` overrides what we are trying to do in the base image (just FYI, this is PR is good to land IMO).

https://github.com/microservices-demo/msd-java/blob/05998875b951dc770ab02859609e5ad13e15b2e7/java.sh#L4

I'm working to update all services to use latest base image, as it's been long overdue and lead to inconsistencies like this `JAVA_OPTS` env var. As a result, the size of final app images should be reduced as we have JRE base image now.

**Pull request merged**